Tests/secrets_white_list.json

```diff
@@ -1913,6 +1913,7 @@
       "384:mzawcK1gOUqJMpMhE5yOaF0O9TwFExCZkUf6H+PknKV+rXkrsH/CxEdubVAjMPW4:AiMVWcK9AhrDNEf"
     ],
     "emails": [
+      "changed-files@v18.7",
```

Comment:
What is this change related to?

Reply:
Because the GitHub Action version string is detected as an email and causing [the validation to fail](https://app.circleci.com/pipelines/github/demisto/content/228267/workflows/1a7c90bd-f45e-4479-b9ac-f942617569d3/jobs/638868). The same was done [here](https://github.com/demisto/content/pull/23306).